### PR TITLE
Assignment-1

### PR DESCRIPTION
**Pull request by Sneha1251**
Screenshot from 2024-05-06 14-45-07
Screenshot from 2024-04-29 12-37-15

### CODE REVIEW

File: src/Assignment-1/lib/math.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+};
+
+const mult = (num1, num2) => {
+  return math.multiply(num1, num2);
+};
+
+const div = (num1, num2) => {
+  return math.divide(num1, num2);
+};
+
```
Comment (DevGarima) [inline src/Assignment-1/lib/math.js]: move lib and utils on root folder -for e.g. src/lib

File: src/Assignment-1/utils/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const fs = require("fs");
+
+const num1 = parseInt(readLine.question("Enter First Number "));
+const num2 = parseInt(readLine.question("Enter Second Number "));
+
+const addResult = add(num1, num2);
+const subResult = sub(num1, num2);
+const multResult = mult(num1, num2);
+const divResult = div(num1, num2);
+
```
Comment (DevGarima) [inline src/Assignment-1/utils/index.js]: move this file to assignment-1 folder

File: src/Assignment-1/utils/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+console.log("hello");
```
Comment (DevGarima) [inline src/Assignment-1/utils/index.js]: remove this console

**Review by DevGarima — CHANGES_REQUESTED** [review]

File: src/Assignment-1/utils/index.js (hunk 3 above)
Comment (Sneha1251) [reply src/Assignment-1/utils/index.js]: done

File: src/Assignment-1/utils/index.js (hunk 2 above)
Comment (Sneha1251) [reply src/Assignment-1/utils/index.js]: done

File: src/Assignment-1/lib/math.js (hunk 1 above)
Comment (Sneha1251) [reply src/Assignment-1/lib/math.js]: done

**Review by DevGarima — APPROVED** [review]

**Pull request merged**